client/src/components/Verejne/entityHelpers.js

```diff
@@ -0,0 +1,16 @@
+// @flow
+import type {Entity} from '../../state'
+
+// entity which is a number is individual, district starts with letter
+export const isIndividual = (eid: string): boolean => {
+  return '0123456789'.indexOf(eid[0]) === -1
+}
+
+export const isPolitician = (entity: Entity): boolean => {
+  // TODO: == is on purpose as this field can be: null, 0, true...
+  return entity.ds && entity.ds[0] == 1
```

Comment:
and what do you check for exactly ? does entity.ds === 0 mean that entity is a politician or not ? what is the case where !!entity.ds does not work ?